src/main/java/com/google/sps/servlets/ChatServlet.java

```diff
@@ -46,31 +47,44 @@
      */
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        String groupID = (String) request.getParameter(GROUP_ID_PROPERTY);
-
-        // This will be a placeholder until we can coordinate how to pass
-        // groupID from page to page
-        if (groupID == null) {
-            groupID = "123";
-        }
+        String groupId = (String) request.getParameter(GROUP_ID_PROPERTY);
```

Comment:
Can you move this to be below where you set groupId for the null/empty scenarios?